server/controllers/finance/reports/vouchers/receipt.handlebars

```diff
@@ -4,6 +4,15 @@
   <header>
     <!-- headings  -->
     <div class="row">
+
+      {{#if details.isCreditNoted}}
+        <div class="text-center" style="position: absolute; border: 5px solid red; -webkit-transform: rotate(10deg); transform: rotate(10deg); left: 35%; top: 20px;">
```

Comment:
It would be better to use `FORM.LABELS.CANCELED` since a voucher can be either a credit or a debit.